.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  extends: [
+    '@mate-academy/eslint-config-react-typescript',
+    'plugin:react/recommended',
+  ],
+  rules: {
+    // React
+    'react/prop-types': 0,
+    'react/self-closing-comp': 0,
+    'react/display-name': 0,
+
+    // JavaScript
+    semi: 0,
+    'no-proto': 0,
+    'no-unused-vars': 0,
```

Comment:
Why did you turn off the unused vars checking? It's a pretty useful thing

Reply:
I read it in https://typescript-eslint.io.
![image](https://user-images.githubusercontent.com/84145664/233474932-bb468ac1-5978-4988-a417-cb6a7d0ab534.png)
https://typescript-eslint.io/rules/no-unused-vars/

Reply:
When I performed the task I used a ready-made template. But now, especially after the cleancode lecture, I'm going to delve deeper into the issue of using linters. And i found this.
![image](https://user-images.githubusercontent.com/84145664/233478012-61c26032-ac37-4d53-bcd8-9b9cfeb69641.png)
https://typescript-eslint.io/rules/no-unused-vars/